app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -56,40 +56,46 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.TELEPHONE'
           <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.telephone}}</p>
           <input type="tel" name="Telephone" value="" formControlName="telephone" *ngIf="!generalSectionButtonsToggle"
             class="app-profile-general-dropdown-input-details app-selected">
+          <mat-error class="app-error-message" *ngIf="profileForm.get('telephone').invalid">
+            {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+          </mat-error>
         </div>
         <mat-divider class="app-general-dropdown-divider"></mat-divider>
         <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.MOBILE' | translate}}</h4>
           <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.mobile}}</p>
         <input type="tel" value="" formControlName="mobile" *ngIf="!generalSectionButtonsToggle"
           class="app-profile-general-dropdown-input-details app-selected">
+        <mat-error class="app-error-message" *ngIf="profileForm.get('mobile').invalid">
+          {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+        </mat-error>
         </div>
       </div>
     </div>
   </div>
 
-  <div class="app-profile-login-row" [formGroup]="profileForm">
-    <div class="app-profile-login" >
+  <div class="app-profile-login-row" [formGroup]="profileForm" hidden>
+    <div class="app-profile-login">
       <div class="app-profile-general-div">
-        <mat-icon class="app-profile-general-icon" (click)="toggleLoginDropdown()">
+        <mat-icon class="app-profile-general-icon" (click)="toggleLoginDropdown()" id="toggle-login-dropdown">
           {{ loginSectionDropdown ?  'expand_more' : 'chevron_right'}}</mat-icon>
         <h4 class="app-general-title">{{'APP.EDIT_PROFILE.LOGIN' | translate}}</h4>
       </div>
 
       <div class="app-general-edit-btn" *ngIf="loginSectionButtonsToggle">
-        <button mat-raised-button (click)="toggleLoginButtons()"
+        <button mat-raised-button (click)="toggleLoginButtons()" id="login-edit-button"
           class="app-general-edit">{{'APP.EDIT_PROFILE.EDIT' | translate}}</button>
       </div>
       <div class="app-general-edit-btn" *ngIf="!loginSectionButtonsToggle">
-        <button mat-raised-button class="app-general-cancel-btn"
+        <button mat-raised-button class="app-general-cancel-btn" id="login-cancel-button"
```

Comment:
this id attribute is not used in this PR?

Reply:
Removed